portafly/doc/ux_specs/backend/edit_backend.md

```diff
@@ -0,0 +1,35 @@
+# Edit Backend
+
+[Back to index](../index.md)
+
+* [Mockups](https://marvelapp.com/prototype/12844cg4/screen/71559182)
+* [Mockups JIRA ticket](https://issues.redhat.com/browse/APPDUX-348)
+* [Parent JIRA ticket](https://issues.redhat.com/browse/THREESCALE-5577)
+
+##### Form
+* Input fields
+  * Name > text input [required]
+    * Validation errors:
+      * Required field on form submit (if left blank)
+      * Name already exists on form submit
+      * Name already exists on _onblur_
+
+  * System name > text input [required] [disabled]
+    * helper text :
+        * System name cannot be changed
+    * NO validation errors
```

Comment:
Is there a reason for the all caps NO? If not, it can be capitalized as `No`

Reply:
@dfennessy emphasize that we don't need validation errors :P

Reply:
Gotcha :smiley: